### ISSUE TITLE
Testing Individual List Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4634]
# Page to Test:

Individual List - https://openlibrary.org/people/mekBot/lists/OL103955L/Childhood_Favorites

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly Book Cover present on this page have empty alt attr, but instead, it should have a meaning full alt attr.
![image](https://user-images.githubusercontent.com/64412143/108602856-7ea0fd80-73ca-11eb-9b63-e210e7c89207.png)


I also wanted to know what should be an ideal alt attribute for book covers cause it's pretty inconsistent across different pages. Some don't have an alt attr; some have empty, some have "Cover of: Book name"

**Comment by bpmcneilly** [comment]
Hey @Yashs911, I think this instance is similar to what I mentioned earlier in #4633 about restructuring links. In most cases, I think generally the Book (or list) name without "Cover of:" would be the best `alt` value if the `img` is by itself.

 However, if we do merge the `img` into another `a` element, like in #4633, then the `alt` can be null as the book's name will already be a part of the link text.

**Comment by Yashs911** [comment]
@bpmcneilly  For this page, I guess it's better to just add alt to the image without merging both links
